static/app/components/modals/inviteMembersModal/renderEmailValue.tsx

```diff
@@ -25,7 +27,9 @@ function renderEmailValue<Option extends OptionTypeBase>(
           {children}
           {!status.sent && !status.error && <SendingIndicator />}
           {status.error && <IconWarning legacySize="10px" />}
-          {status.sent && <IconCheckmark legacySize="10px" color="success" />}
+          {status.sent && !organization.features.includes('invite-members-new-modal') && (
+            <IconCheckmark legacySize="10px" color="success" />
```

Comment:
this is for an edge case where a user successfully invites and email and then inputs the same email again: we don't want a checkmark to appear suggesting that the email has successfully been sent, because when the user clicks "send" the second time the invite will fail to send (since the member already exists)

Reply:
🤔 i wonder if there's something we should be looking at like request state here. something like: "If it's still pending form submission, then ..."